WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/ShippingLabelFormViewController.swift

```diff
@@ -546,6 +556,55 @@ private extension ShippingLabelFormViewController {
     }
 }
 
+// MARK: - Top Banner
+//
+private extension ShippingLabelFormViewController {
+    /// Creates a Top Banner View containing the EU Shipping Notice.
+    ///
+    func showTopBannerView() {
+        guard viewModel.shouldDisplayShippingNotice else {
+            return
+        }
+
+        let topBannerView = EUShippingNoticeTopBannerFactory.createTopBanner(
+            onDismissPressed: {
+                self.hideTopBannerView()
+            },
+            onLearnMorePressed: { instructionsURL in
+                self.presentShippingInstructionsView(instructionsURL: instructionsURL)
+            })
```

Comment:
Since these are `@escaping` closures, it's safer to capture `self` weakly to avoid possible retain cycles.

Reply:
Oh, true! I keep forgetting about this 😅, thanks! Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/9636/commits/1069ba58b8a9328af623621a3c2ead17896b83c7